tripal/tests/src/Unit/TripalDbx/TripalDbxUnitTest.php

```diff
@@ -45,24 +46,91 @@
 class TripalDbxUnitTest extends UnitTestCase {
 
   /**
-   * Test members.
+   * Prophesize config factory object.
    *
-   * "pro*" members are prophesize objects while their "non-pro*" equivqlent are
-   * the revealed objects.
+   * "pro*" members are prophesize objects while their "non-pro*" equivalent
+   * are the revealed objects.
+   *
+   * @var Prophecy\Prophecy\ObjectProphecy
+   */
+  protected ObjectProphecy $proConfigFactory;
+
+  /**
+   * Revealed config factory object.
+   *
+   * @var object
+   */
+  protected object $configFactory;
+
+  /**
+   * Prophesize config object.
+   *
+   * @var Prophecy\Prophecy\ObjectProphecy
+   */
+  protected ObjectProphecy $proConfig;
+
+  /**
+   * Revealed config object.
+   *
+   * @var object
+   */
+  protected object $config;
+
+  /**
+   * Prophesize container object.
+   *
+   * @var Prophecy\Prophecy\ObjectProphecy
+   */
+  protected ObjectProphecy $proContainer;
+
+  /**
+   * Revealed container object.
+   *
+   * @var object
+   */
+  protected object $container;
+
+  /**
+   * Prophesize connection object.
+   *
+   * @var Prophecy\Prophecy\ObjectProphecy
+   */
+  protected ObjectProphecy $proConnection;
+
+  /**
+   * Revealed connnection object.
```

Comment:
Let's be clear here that we are not prophesizing the TripalDbxConnection but rather the Drupal Connection.

```suggestion
   * Prophesize DRUPAL connection object.
   *
   * @var Prophecy\Prophecy\ObjectProphecy
   */
  protected ObjectProphecy $proConnection;

  /**
   * Revealed DRUPAL connnection object.
```